data/python/utils.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import platform
+
+import jep
+import ghidrathon
+from java.lang import System
+
+ALLOWED_EXCEPTIONS = [RuntimeError, OSError]
+
+
+def log_env_details(exc_type):
+    if any(issubclass(exc_type, exc_class) for exc_class in ALLOWED_EXCEPTIONS):
+        print(
+            "Python={python_version}, Arch={arch}, OS={os}, Ghidra={ghidra_version}, Java={java_version}, Ghidrathon={ghidrathon_version}, Jep={jep_version}".format(
+                python_version=platform.python_version(),
+                arch=System.getProperty("os.arch"),
+                os=System.getProperty("os.name"),
+                ghidra_version=getGhidraVersion(),
+                java_version=System.getProperty("java.version"),
+                ghidrathon_version="4.0.0",
```

Comment:
Hi @mike-hunhoff, 

I am not sure how I can call `GhidrathonPlugin` class method `getVersion` in here?

<details>
<summary>I brute-forced my way into calling getVersion, via the builtin interpreter</summary>

![grafik](https://github.com/mandiant/Ghidrathon/assets/31004043/8b565f2f-37b0-4dc0-8e02-fd18d44d9a77)

</details>

Reply:
After installing the Ghidrathon extension you should be able to do something like:

```python
import ghidrathon
print(f"version: {ghidrathon.GhidrathonPlugin.getVersion()}")
```

Reply:
This assumes `static` addition to `getVersion` as mentioned below.